tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -13,25 +13,30 @@ ${PUBLICATION_NAME}  UI tests - publish release and amend 2 %{RUN_IDENTIFIER}
 ${SUBJECT_NAME}  Seven filters
 ${SECOND_SUBJECT}  upload file test
 ${THIRD_SUBJECT}  upload file test with filter subject
-${METHODOLOGY_NAME}  UI tests - publish release and amend 2 %{RUN_IDENTIFIER}
-${METHODOLOGY_TITLE}  ${METHODOLOGY_NAME} - Title
 
 *** Test Cases ***
-Navigate to manage methodoligies page
+Create publication
```

Comment:
I think you should be able to the `user creates publication` keyword here (from `admin-common.robot`)

Reply:
Done!